pages/services.tsx

```diff
@@ -125,3 +129,28 @@ const Services = () => {
 }
 
 export default Services
+
+export async function getServerSideProps() {
+  try {
+    const response = await axios.get(
+      'https://www.srmkzilla.net/api/instagram',
```

Comment:
Why would you call the api itself? Why not just write the logic you wrote in /api here? both run on server

Reply:
done